From d9b4541f726965344fdc28457a096c8d89a43f6d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=E1=B4=80=E1=B4=8D=E1=B4=80=CA=80=20V=C9=AA=EA=9C=B1?=
 =?UTF-8?q?=E1=B4=98=E1=B4=9C=E1=B4=9B=E1=B4=87?=
 <samarvispute121@gmail.com>
Date: Wed, 19 Feb 2020 18:20:20 +0530
Subject: [PATCH 07/25] Enable latch_unsignaled on exynos devices

ro.hardware=exynos* on some samsung devices
eg. Samsung Galaxy M20
---
 rw-system.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rw-system.sh b/rw-system.sh
index e53acf3..db4efb2 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -305,7 +305,7 @@ if busybox_phh unzip -p /vendor/app/ims/ims.apk classes.dex | grep -qF -e Landro
     mount -o bind /system/phh/empty /vendor/app/ims/ims.apk
 fi
 
-if getprop ro.hardware | grep -qF samsungexynos; then
+if getprop ro.hardware | grep -qF samsungexynos -e exynos; then
     setprop debug.sf.latch_unsignaled 1
 fi
 
-- 
2.17.1

